src/BucketListItem.jsx

```diff
@@ -1,28 +1,42 @@
 export function BucketListItem(props) {
-  return (
-    <li
-      class="list-item"
-      style={{
-        "text-decoration": props.item.complete ? "line-through" : undefined,
-      }}
-    >
-      <label>
-        <input
-          type="checkbox"
-          checked={props.item.complete}
-          onChange={() => {
-            props.setItems((items) => {
-              const newItems = items.map((item) =>
-                props.item === item
-                  ? { ...item, complete: !item.complete }
-                  : item
-              );
-              return newItems;
+    const { item, setItems } = props;
+
+    const handleDelete = () => {
+        setItems((prevItems) => {
+            return prevItems.filter((prevItem) => {
+                return prevItem.text !== item.text;
             });
-          }}
-        />
-        {props.item.text}
-      </label>
-    </li>
-  );
+        });
+    };
+
+    return (
+        <li
+            class="list-item"
+            style={{
+                "text-decoration": props.item.complete
+                    ? "line-through"
+                    : undefined,
+            }}>
+            <label>
+                <input
+                    type="checkbox"
+                    checked={props.item.complete}
+                    onChange={() => {
+                        props.setItems((items) => {
+                            const newItems = items.map((item) =>
+                                props.item === item
+                                    ? { ...item, complete: !item.complete }
+                                    : item
+                            );
+                            return newItems;
+                        });
+                    }}
+                />
+                {props.item.text}
+            </label>
+            <button onClick={handleDelete} style={{ "margin-left": "10px" }}>
```

Comment:
This issue doesn't require the delete wish functionality. Delete wish is a separate issue and in open source project management we follow one issue vs one pull request.

Please remove the code for delete issue and only fix the issue asked for.